src/content/docs/experiment/link-element.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Monetization <link> element
+tableOfContents:
+    maxHeadingLevel: 4
+---
+
+import { LinkOut, Hidden } from "@interledger/docs-design-system";
+import Specification from '/src/components/docs/Specification.astro'
+import BrowserCompat from '/src/components/docs/BrowserCompat.astro'
+import data from '/src/data/browser-compat-data/link.json'
+import { Steps, Tabs, TabItem, Badge } from '@astrojs/starlight/components';
+
+<Badge text="HTML DOM API" style="position: relative; top:-30px;" />
+
+From a creator/developer perspective, the monetization `<link>` element is one of the two key pieces to web monetizing a page. The second piece is a wallet address/payment pointer.
+
+## Prerequisites
+
+To web monetize an HTML document, such as a web page:
+* You must have an account with a [compatible wallet](/resources/wallets)
+* You must have the wallet address, or payment pointer in <LinkOut href="https://paymentpointers.org">URL format</LinkOut>, from your wallet provider
+* Your document must be secure and served over HTTPS
+
+## Monetization `<link>` element
+
+The monetization `<link>` element is what indicates your HTML page supports Web Monetization.
+
+### Syntax
+
+```html
+<link rel="monetization" href="https://(paymentUrl)">
+```
+
+* The `rel` attribute is always `monetization`
+* The `href` attribute equals your wallet address or your payment pointer in URL format
+
+For example:
+
+```html
+<link rel="monetization" href="https://wallet.example.com/alice">
+```
+
+### Placement
+
+A monetization `<link>` is `body-ok`, meaning it's allowed in your page's `<head>` and/or `<body>`.
+
+### Multiple monetization links
+
+An HTML page can contain multiple monetization `<link>` elements; however, there are a few factors that determine if and how a multiple link scenario is monetized. 
+
+First, most browsers enforce a maximum limit of six parallel HTTP connections to the same server. When the maximum is reached, the browser will wait until one of the existing HTTP connections is closed before it initiates the next connection.
```

Comment:
Is this still relevant with HTTP2 and is the connection limit really relevant to WM? For example, delay in loading wallet info is not as big issue as delay in loading stylesheets/scripts.
Also, it depends on when/where the browser is sending request from (like multiple tabs opening won't respect 6 connection limit?)

Reply:
Running this by Alex, this info is from him

Reply:
Agreed that it's less relevant for http2. Will update.

Reply:
Additional feedback for the history...

it used to be that browsers counted URLs in http1, so for example a page accessing `example.org/1.png`, `example.org/2.png` and `example.org/3.png` was using 3 of those connection slots. with http2, it only counts `example.org`, so all those 3 URL use a single http connection. a good website won't ideally need to access 6 domains for loading stuff. iirc, the 6 connection limit is per tab but I'm not sure about that